src/content/docs/en/reference/cli-reference.mdx

```diff
@@ -68,7 +68,7 @@ Commands
              docs  Open documentation in your web browser.
              info  List info about your current Astro setup.
           preview  Preview your build locally.
-             sync  Generate content collection types.
+             sync  Generates TypeScript types for all Astro modules.
```

Comment:
Shouldn't this be `Generate` instead to be consistent with the other options? I saw this is the same description inside the code, and it's a trivial one-line change.